install/amazon_rds/_04_setting_env_vars_multi.mdx

```diff
@@ -0,0 +1,40 @@
+import { APIKeyInstructions } from './04_configure_the_collector_ec2.mdx'
+
+export const CollectorDotEnv = ({ apiKey }) => {
+  return (
+    <CodeBlock>
+      {`CONFIG_CONTENTS="[pganalyze]
+api_key = ${apiKey ?? 'your_pga_organization_api_key'}\n
+[server1]
+db_host = instance-id1.account-id.us-east-1.rds.amazonaws.com
+db_name = your_database_name
+db_username = your_monitoring_user
+db_password = your_monitoring_user_password
+db_sslrootcert = rds-ca-global
+db_sslmode = verify-full\n
+[server2]
+db_host = instance-id2.account-id.us-east-1.rds.amazonaws.com
+db_name = your_database_name
+db_username = your_monitoring_user
+db_password = your_monitoring_user_password
+db_sslrootcert = rds-ca-global
+db_sslmode = verify-full"`}
+    </CodeBlock>
+  )
+}
+
+<CollectorDotEnv />
+
+This multiline variable definition is **only supported by Docker Compose and not by the Docker CLI**.
+If you're using the Docker CLI (e.g. `docker run --env-file pganalyze_collector.env`),
+please replace new lines with `\n`, like `CONFIG_CONTENTS="[pganalyze]\napi_key = ..."`.
+Alternatively, run one pganalyze collector Docker container for each server
+using the Single Server method.
```

Comment:
[note for reviewer] This is something I actually should be double checking by myself, but looking the following places, I think this is the current status.
https://github.com/containers/podman/issues/18724#issuecomment-1757162444
https://github.com/docker/cli/blob/master/opts/envfile.go

Reply:
Uhmm I tested this now but sadly this doesn't work. We're doing `echo "$CONFIG_CONTENTS"` but this doesn't handle new lines (`\n`)

https://github.com/pganalyze/collector/blob/32b62a13766663ed1167022cbfa41d2e1c89fca6/contrib/docker-entrypoint.sh#L6

```shell
$ CONFIG_CONTENTS="[pganalyze]\napi_key = test_key"
$ echo $CONFIG_CONTENTS 
[pganalyze]\napi_key = test_key
$ echo -e $CONFIG_CONTENTS 
[pganalyze]
api_key = test_key
```

There are two options:
1. add `-e` in the docker-entrypoint.sh and make a new release, then ship this doc
2. just say "If you're using the Docker CLI, it won't work, sorry"

... actually, this makes me wonder if Docker Compose way even works. With Docker Compose, we can define config var with new line without doing `\n` join, though at the end, what we do is `echo "$CONFIG_CONTENTS"` and if that part doesn't handle new line, I don't think it'll work 🤔